modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/repositories/workflowcatalog/WorkflowRepository.java

```diff
@@ -17,16 +17,13 @@
  * specific language governing permissions and limitations
  * under the License.
  *
-*/
+ */
 package org.apache.airavata.registry.core.repositories.workflowcatalog;
 
-import org.apache.airavata.model.WorkflowModel;
-import org.apache.airavata.model.application.io.OutputDataObjectType;
-import org.apache.airavata.registry.core.entities.workflowcatalog.WorkflowEntity;
-import org.apache.airavata.registry.core.utils.DBConstants;
-import org.apache.airavata.registry.core.utils.ObjectMapperSingleton;
-import org.apache.airavata.registry.core.utils.QueryConstants;
-import org.apache.airavata.registry.core.utils.WorkflowCatalogUtils;
+import org.apache.airavata.model.commons.airavata_commonsConstants;
```

Comment:
Why not AiravataCommonsConstants ?

Reply:
@DImuthuUpe It's an old thrift auto generated file. Need to change the .thrift filename along with few modifications to the related usages, in order to fix this. Is it OK to do it in another PR?

Reply:
@yasgun It's better to send it as a separate PR however let's keep it as it is for the instance.